src/components/MyWorkplace/ChangeEmail.tsx

```diff
@@ -31,8 +31,8 @@ const ChangeEmail = () => {
 
     const onSubmit = handleSubmit(async data => {
             await dispatch(changeEmailAuth(data.currentEmail, data.newEmail, data.password, role))
-            if (role == "USER") history.push(`/MyOffice/${id}`)
-            else if (role == "MASTER") history.push(`/MyOffice/${id}`)
+            if (role === "USER") history.push(`/MyOffice/${id}`)
```

Comment:
Create ENUM ROLES
I guess we talk about it in the past

Reply:
we talked about it on the backend. I didn't have time to change it on the front. corrected now